doc/tutorial/1_main_climada.ipynb

```diff
@@ -668,8 +668,7 @@
    "source": [
     "from climada.entity.exposures import LitPop\n",
     "\n",
-    "exp_litpop = LitPop()\n",
-    "exp_litpop.set_country('Puerto Rico', res_arcsec = 120)  # We'll go lower resolution than default to keep it simple\n",
+    "exp_litpop = LitPop.from_countries('Puerto Rico', res_arcsec = 120)  # We'll go lower resolution than default to keep it simple\n",
```

Comment:
Don't forget to use the new constructor method in all tutorials, including Jupyter notebooks. You can search through `ipynb` files using the `grep` shell command.